### PR TITLE
chore: enable i18n lint

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/demo/src/modules/pipes/field-error/examples/5/index.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ export default class Example {
 
     constructor() {
         this.form = this.fb.group({
+            // eslint-disable-next-line i18n/no-russian-character
             text: ['русский текст', Validators.required],
```
Comment (splincode) [inline projects/demo/src/modules/pipes/field-error/examples/5/index.ts:46]: image

File: package.json [hunk 2]
```diff
@@ -80,10 +80,6 @@
             "plugin:@taiga-ui/experience/entry-points"
         ],
         "rules": {
-            "i18n/no-greek-character": "off",
```
Comment (waterplea) [inline package.json:83]: Russian I understand, so there's no Russian "c" hidden in code by accident, but what's wrong with others?

Comment (splincode) [reply package.json:83]: `but what's wrong with others`

prevent contribution like this

image
image

Comment (splincode) [reply package.json:83]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**